### PR TITLE
display new status Debugging when server is started in debug mode (#333)

### LINKED ISSUE #333: Should be able to see that the server is currently debugging
Steps:

1. ASSERT: have a stopped server
1. EXEC: pick "Debug" in the ctx menu

Result:
server turns (Starting) and then (Started) once it's all up. I have no hint that it's currently debugging 

Expected result:
I should be able to see that the server is currently debugging and would hit breakpoints that I set in my code.

### PR DESCRIPTION
**Pull request by lstocchi**
it fixes #333

### CODE REVIEW

File: src/serverExplorer.ts [hunk 1]
```diff
@@ -346,7 +346,9 @@ export class ServerExplorer implements TreeDataProvider< Protocol.ServerState |
             const state: Protocol.ServerState = item as Protocol.ServerState;
             const handle: Protocol.ServerHandle = state.server;
             const id1: string = handle.id;
-            const runState: string = this.runStateEnum.get(state.state);
+            const runState: string = (state.state === 2 && state.runMode === 'debug') ?
```
Comment (adietish) [inline src/serverExplorer.ts]: wouldn't it be more consistent to call this variable ```runMode```?

File: src/serverExplorer.ts [hunk 2]
```diff
@@ -346,7 +346,9 @@ export class ServerExplorer implements TreeDataProvider< Protocol.ServerState |
...
+                                    'Debugging' :
```
Comment (adietish) [inline src/serverExplorer.ts]: I believe that we should have a test that asserts the fact that if the runMode is 'debug' the server label would show 'Debugging'. Changing this label in an inconspicuous move breaks the command visibility and no one would notice for a moment.

File: src/serverExplorer.ts (hunk 1 above)
Comment (adietish) [inline src/serverExplorer.ts]: * ```state.runMode === 'debug'``` should use the constant ```state.runMode === ServerState.RUN_MODE_DEBUG```.
* ```state.state === 2``` should better be ```state.state === ServerState.STARTED```

Comment (lstocchi) [reply src/serverExplorer.ts]: well, not sure. runState can also have as a value "stopped" so if we change the name to runMode is like we're telling ... in which mode the server is running? and it doesn't make sense if the server is stopped. I think it's better runState or serverState. What do you think??

**Pull request merged**